pokemonterminal/terminal/adapters/conemu.py

```diff
@@ -10,17 +10,16 @@ class ConEmuProvider(_TProv):
     def is_compatible() -> bool:
         return "CONEMUPID" in os.environ
 
+    def __run_command(command: str):
+        output = subprocess.check_output(f'ConEmuC -GuiMacro {command}', shell=True).decode(sys.stdout.encoding)
+        if output != 'OK':
+            print(output)
+
     def change_terminal(path: str):
-        # ConEmu has issues with the quoting when using an arg list, so use shell=True
-        # ConEmu requires the image path to have escaped slashes. Enforce this by converting all slashes in the path to
-        # backlashes, replace all previously doubled slashes with a single slash, then replace all single slashes with
-        # double backslash
-        cmd = 'ConEmuC -GuiMacro SetOption("Background Image", "{}")'.format(os.path.normpath(path).replace(
-            '\\\\', '\\').replace('\\', '\\\\'))
```

Comment:
I seem to recall running into weird issues with image paths and needing escaped slashes in the paths. If it works without that's great because it's much cleaner, but maybe do a quick doublecheck that this doesn't somehow cause the issue with the extra images

Reply:
Haven't encountered issues by myself, except for extra. Extra images do work when reverting to master. But the after a few debug prints, the only difference between master and my PR is yours using double slashes. Really weird.

I'll add back the double backslashes to my PR and see if they work.

Reply:
Weird, I don't get it either...

Reply:
I think what happens is that ConEmuC supports its own escaping, and that `\E` is a valid escape code for it. `\\` prevents it because it escapes the backslash.